### PR TITLE
First readme pass, with questions to be answered.

### PR DESCRIPTION
**Pull request by Bjohnson131**
This fixes issue [#9](https://github.com/manutara-org/platform/issues/9).

## Changes:

Added various markdown files to the project.

## Checklist:

 - [x] PR attached to an issue.
 - [ ] New doc-tests added. 
 - [x] All tests pass.
 - [x] All linting passes.
 - [x] All build steps pass.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+-----
+- who are we allowing contributions from?
+- who can submit a PR?
+- what criteria will we use to evaluate a PR?
+
+### Development
+To set up a development environment for the Manutara Platform is simple. Given that you have Rust and GCC already installed, you can 
+follow the steps below.
+ 1. Fork and clone Manutara.
+ 2. Make sure `glibc`, and `openssl` development libraries are installed on your system.
```
[the remark this answers is not in the document]
Comment (Bjohnson131) [reply readme.md:38]: ^ rustup is great, but the intentionally excludes GLibc. For OpenSSL, this is probably the way.

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+ 3. Build using the `cargo build` command to verify your environment.
+ 4. Happy coding!
+
+# Below this line are unimplemented features.
```
[the remark this answers is not in the document]
Comment (Bjohnson131) [reply readme.md]: Moved to readme.md.todo. this file will be updated a lot in the coming days.

**Comment by Bjohnson131** [comment]
Marking as Ready for Review, still needs some legal items taken care of.

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+## Manutara - platform
+### A data-first, event-driven microservice communication platform using [Event Driven Architecture](https://martinfowler.com/articles/201701-event-driven.html) concepts.
+
+[Aaron I need you to make a logo for this like you did with DECKS and KAHM.]
```
Comment (spiegela) [inline readme.md:4]: Let's remove the note for now, but I'll definitely start working on it ;-)

```suggestion
```

**Review by spiegela — APPROVED** [review]

File: readme.md (hunk 3 above)
Comment (Bjohnson131) [reply readme.md:4]: Deal!

File: pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+_Describe your changes here_
+
+## Checklist:
+
+ - [ ] PR attached to an issue.
+ - [ ] New doc-tests added. 
+ - [ ] All tests pass.
+ - [ ] All linting passes.
+ - [ ] All build steps pass.
```
[the remark this answers is not in the document]
Comment (Bjohnson131) [reply pull_request_template.md:13]: I think I'm going to add this anyway, for completeness- doesn't hurt to have, right?

**Pull request merged**